mods/gen/rules/structures.yaml

```diff
@@ -3302,6 +3303,18 @@ building.internet_center:
 		LocalUpgrades: upgrade.land_mines, upgrade.neutron_mines
 	ActorStatOverride@ZH_UNITS:
 		Upgrades: upgrade.satellite_hack1, upgrade.satellite_hack2, upgrade.land_mines, upgrade.neutron_mines
+	FreePassenger@AI_1:
+		Actor: infantry.hacker
+		RequiresCondition: AImicroManage
+	FreePassenger@AI_2:
+		Actor: infantry.hacker
+		RequiresCondition: AImicroManage
+	FreePassenger@AI_3:
+		Actor: infantry.hacker
+		RequiresCondition: AImicroManage
+	FreePassenger@AI_4:
+		Actor: infantry.hacker
+		RequiresCondition: AImicroManage
```

Comment:
AI should get Super Hackers if they are Infantry General.

Reply:
My thought is give 20 cash tick for internet center, basically equal to one black market of GLA, which should be fair to other factions AI player.

Also it is easy on maintaining so I am lazy here.